commit 3ba21538ddc7b7e892b2786cb3664a354c8cd01d (HEAD -> master)
Author: Jerry Jelinek <jerry.jelinek@joyent.com>
Date:   2019-10-07T08:14:55-06:00 (9 hours ago)
    
    TRITON-1911 sdc-server support for custom zpool width
    Reviewed by: Pedro Palazón Candel <pedro@joyent.com>
    Approved by: Pedro Palazón Candel <pedro@joyent.com>

diff --git a/tools/bin/sdc-server b/tools/bin/sdc-server
index 87d511dd..a288f536 100755
--- a/tools/bin/sdc-server
+++ b/tools/bin/sdc-server
@@ -83,6 +83,7 @@ function setup_usage()
         '',
         '    hostname=<string>                                 - Server hostname',
         '    spares=<0-N>                                      - disk spares',
+        '    width=<1-N>                                       - stripe width',
         '    cache=<true|false>                                - disk cache',
         '    layout=<single|mirror|raidz1|raidz2|raidz3>       - disk layout',
         '    postsetup_script=<string>                         - script to be run right after server setup',
@@ -1554,6 +1555,8 @@ function dispatchCommand(command, args, callback)
                     setupOpts.params.disk_spares = param[1];
                 } else if (param[0] === 'cache') {
                     setupOpts.params.disk_cache = param[1];
+                } else if (param[0] === 'width') {
+                    setupOpts.params.disk_width = param[1];
                 } else if (param[0] === 'layout') {
                     setupOpts.params.disk_layout = param[1];
                 } else {
